src/appComponents/tracks/TracksController.js

```diff
@@ -1,100 +1,119 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
+/* exported error */
 
-var tracksModule = angular.module('oe.tracks', ['ui.router']);
+const tracksModule = angular.module("oe.tracks", ["ui.router"]);
+
+tracksModule.config(["$stateProvider", function($stateProvider) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier '$stateProvider' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2418442828/issues/source?bid=3233857&fileBranchId=3237740#l8)